src/opower/utilities/nationalgridnymetro.py

```diff
@@ -0,0 +1,23 @@
+"""National Grid NY Metro."""
+
+from .base import UtilityBase
+from .nationalgrid import NationalGrid
+
+
+class NationalGridNYMetro(NationalGrid, UtilityBase):
+    """National Grid NY Metro."""
+
+    @staticmethod
+    def name() -> str:
+        """Distinct recognizable name of the utility."""
+        return "National Grid (NY Metro)"
+
+    @staticmethod
+    def subdomain() -> str:
+        """Return the opower.com subdomain for this utility."""
+        return "ngny-gas"
```

Comment:
Is there a different domain for electricity?

Reply:
Not that I'm aware of - I only use national grid for gas